fuzz/travis-fuzz.sh

```diff
@@ -8,9 +8,17 @@ if [ ${incorrectFilenames} -gt 0 ]; then
 	exit 2
 fi
 
+if [ "$1" == "" ]; then
+	TARGETS=fuzz_targets/*
+else
+	TARGETS=fuzz_targets/"$1".rs
+fi
+
 # Testing
 cargo install --force honggfuzz --no-default-features
-for TARGET in fuzz_targets/*; do
+for TARGET in $TARGETS; do
+	echo $TARGET
+	continue
```

Comment:
Is this even doing the fuzzing? Won't the `continue` just skip rest of the loop

Reply:
lol oops this was test code